balancer.go

```diff
@@ -142,3 +147,55 @@ func (r *balancer) Perform(s Strategy) map[string]map[string][]int32 {
 	}
 	return res
 }
+
+func (r *balancer) performMultibalance(strat Strategy) map[string]map[string][]int32 {
+	members := make(map[string]struct{}, 1)
+	topics := make(map[string][]int32, 1)
+	for topic, info := range r.topics {
+		topics[topic] = info.Partitions
+		for _, memberID := range info.MemberIDs {
+			members[memberID] = struct{}{}
+		}
+	}
+
+	topicNames := make([]string, 0)
+	for topic, _ := range topics {
+		topicNames = append(topicNames, topic)
+	}
+	sort.Strings(topicNames)
+
+	memberIDs := make([]string, 0)
```

Comment:
ou know the required cap, avoid reallocations and pls use make([]string, 0, len(members))